app/routes/data-quality.ts

```diff
@@ -1,3 +1,191 @@
+          const agendaItems: AdapterPopulatedRecordArrayWithMeta<AgendaItemModel> =
+            await this.store.query('agenda-item', {
+              filter: {
+                sessions: {
+                  'governing-body': {
+                    'administrative-unit': {
+                      location: {
+                        label: params.municipalityLabel || undefined,
+                      },
+                    },
+                    classification: {
+                      label: governingBody.label,
+                    },
+                  },
+                },
+              },
+              size: 1,
+            });
+
+          return {
+            classification: governingBody.label,
+            count: getCount(agendaItems) || 0,
+          };
+        });
+
+      // resolve the promises
+      agendaItemsPerGoverningBodyClassification = await Promise.all(
+        agendaItemsPromises
+      );
+
+      agendaItemsPerGoverningBodyClassification =
+        agendaItemsPerGoverningBodyClassification.map((item) => ({
+          ...item,
+          percentage:
+            ((item.count || 0) / (getCount(totalCountAgendaItems) || 1)) * 50,
+        }));
+
+      agendaItemsPerGoverningBodyClassification.push({
+        classification: 'Totaal',
+        count: getCount(totalCountAgendaItems) || 0,
+        percentage: 50,
+      });
+
+      // get the total count of agenda items which have a vote or voters
+
+      const agendaItemsWithVoteOrVoters: AdapterPopulatedRecordArrayWithMeta<VoteModel> =
+        await this.store.query('vote', {
+          'filter[:or:][handled-by][subject][sessions][governing-body][is-time-specialization-of][administrative-unit][location][label]':
+            params.municipalityLabel || undefined,
+          'filter[:or:][handled-by][subject][sessions][governing-body][administrative-unit][location][label]':
+            params.municipalityLabel || undefined,
+          size: 1,
+        });
+
+      return {
+        totalCountAgendaItems: getCount(totalCountAgendaItems) || 0,
+        percentageAgendaItemsTreatedToTotal: Math.round(
+          ((getCount(totalCountAgendaItemsTreated) || 0) /
+            (getCount(totalCountAgendaItems) || 1)) *
+            100
+        ),
+        percentageAgendaItemsWithTitleAndDescriptionToTotal: Math.round(
+          ((getCount(totalCountAgendaItemsWithTitleAndDescription) || 0) /
+            (getCount(totalCountAgendaItems) || 1)) *
+            100
+        ),
+        agendaItemsPerGoverningBodyClassification:
+          // first sort by label then sort by count descending set total at the begining of the array
+          agendaItemsPerGoverningBodyClassification
+            .sort((a, b) =>
+              (a.classification || '').localeCompare(b.classification || '')
+            )
+            .sort((a, b) => (b.count || 0) - (a.count || 0))
+            .sort((a, b) => (a.classification === 'Totaal' ? -1 : 0)),
```

Comment:
**Blocker:** Chaining `sort` calls will not have the effect that you describe in the comment. Here only the latest sort() will have effect. To do what it expected, you have to merge all code in one `sort` method. You can find an example of multicriterial sorting here: https://dev.to/markbdsouza/js-sort-an-array-of-objects-on-multiple-columns-keys-2bj1

Reply:
@aliokan I've resolved everything except for the sort. I can't get it to work the way it does currently using multicriterial sorting :/